src/update.rs

```diff
@@ -83,6 +83,21 @@ pub fn update_bugzilla(
     Ok(profile)
 }
 
+pub fn update_slack(
+    slack_uri: String,
+    slack_username: String,
+    mut profile: Profile,
+    store: &SecretStore,
+) -> Result<Profile, Error> {
+    let now = &Utc::now().to_rfc3339_opts(SecondsFormat::Secs, true);
+    let uris_kv_pairs = vec![(
```

Comment:
This discards if slack was previously shown in ContactMe. We could look at the existing fields to keep it. But that's optional. Just wanted to mention it here.

Reply:
I don't know what you mean here.

Reply:
I think Florian is referring to the method by which we store whether slack should be shown in the "contact me" dropdown at the top of the profile:

https://github.com/mozilla-iam/dino-park-front-end/blob/5060d4ff072c00a143b4b822c83390dae23ef698/src/components/_mixins/AccountsMixin.vue#L89-L94

The `#y` or `#n` at the end determines whether it is or isn't shown (respectively), but this currently forces it to not be shown (even if it was shown before).

Reply:
Ah yes, this makes sense. Thank you!